modules/simple_bucket/variables.tf

```diff
@@ -211,3 +211,18 @@ variable "internal_encryption_config" {
   })
   default = {}
 }
+
+variable "ip_filter" {
+  description = "The IP filter configuration for the bucket."
```

Comment:
Can we add more details in the description as to how to provide values for this?
Can `public_network_source` and `vpc_network_sources` be configured together? If not, can we add in the description?